tests/Core/Tokenizers/Tokenizer/RecurseScopeMapDefaultKeywordConditionsTest.php

```diff
@@ -158,7 +158,7 @@ public function testSwitchDefault($testMarker, $openerOffset, $closerOffset, $co
         if (($opener + 1) !== $closer) {
             $end = $closer;
             if (isset($conditionStop) === true) {
-                $end = $conditionStop;
+                $end = ($opener + $conditionStop);
```

Comment:
```suggestion
                $end = ($token+ $conditionStop);
```

All the other offsets are in relation to the `testMarker`, so this one should be too, otherwise things get really confusing.

The docblock should probably also mention this:
```diff
-     * @param int|null $conditionStop The expected offset at which tokens stop having T_DEFAULT as a scope condition.
+     * @param int|null $conditionStop The expected offset in relation to the testMarker, at which tokens stop
+     *                                having T_DEFAULT as a scope condition.
```

Reply:
Done